lib/ansible/modules/storage/infinidat/infini_vol.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Gregory Shulov (gregory.shulov@gmail.com)
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: infini_vol
+version_added: 2.3
+short_description:  Create, Delete or Modify volume on Infinibox
+description:
+    - This module creates, deletes or modifies volume on Infinibox.
+options:
+  name:
+    description:
+      - Volume Name
+    required: true
+  state:
+    description:
+      - Creates/Modifies volume when present or removes when absent
+    required: false
+    default: present
+    choices: [ "present", "absent" ]
+  size:
+    description:
+      - Volume size in MB, GB or TB units. See examples.
```

Comment:
The example uses `1T`, 

However later on you have
`module.fail_json(msg='size (Physical Capacity) should be defined in MB, GB, TB or PB units')`